django/utils/feedgenerator.py

```diff
@@ -223,7 +224,7 @@ def latest_post_date(self):
                     if latest_date is None or item_date > latest_date:
                         latest_date = item_date
 
-        return latest_date or datetime.datetime.now()
+        return latest_date or datetime.datetime.utcnow().replace(tzinfo=utc)
```

Comment:
Tim's point is to make sure a regression test would fail if someone inadvertently refactored the code to use `timezone.now()` instead. I suggest also adding a comment above explaining the rationale for not using `timezone.now()`.